magento/magento2
# Qty box visibility issue in wishlist when product is out of stock

### Description (*)

Magento 2.3.x
Magento 2.2.x

Qty box should not be displayed in the wishlist if an added product is out of stock.

### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title
2. ...

### Manual testing scenarios (*)

1. Set out of stock product display as a front end from admin configuration by below steps
Stores >> Setting >> Configuration >> Catalog >> Inventory >> Display Out of Stock Products	 set Yes
2. Login as a customer in frontend
3. Add out of stock product into wishlist
4. Review the product in My Wish List section in frontend where qty box is displayed for out of stock product.
![Screenshot from 2019 04 17 11 50 01](https://user-images.githubusercontent.com/43400051/56265739-f7b4d700-6107-11e9-8b82-44bae54be820.png)

### Contribution checklist (*)
